types/tslint.json

```
@@ -0,0 +1,3 @@
+{
+  "extends": "dtslint/dt.json"
```

this causes the error `Error: If 'tslint.json' is present, it should extend "dtslint/dtslint.json"`

```suggestion
  "extends": "dtslint/dtslint.json"
```